### PR TITLE
API skeleton for group 1

### PR DESCRIPTION
**Pull request by felixschorer**
Endpoints for path `/entities`

### CODE REVIEW

**Comment by kordianbruck** [comment]
@MusicConnectionMachine/group-1 please review this!

**Comment by ShilpaGhanashyamGore** [comment]
On it ! I will fork this repository and port the API implementation.

**Comment by kordianbruck** [comment]
@ShilpaGhanashyamGore please first review this PR and approve if everything is up to specs!

Docs: https://help.github.com/articles/about-pull-request-reviews/

File: api/swagger/swagger.yaml [hunk 1]
```diff
@@ -20,7 +20,124 @@ produces:
...
+          in: body
+          description: entity data
+          required: true
+          schema:
+            $ref: '#/definitions/UpdateEntity'
+      responses:
+        "200":
+          description: Success
+          schema:
+            $ref: '#/definitions/Entity'
```
Comment (ShilpaGhanashyamGore) [inline api/swagger/swagger.yaml]: Does this mean we return the updated data ?
If so I don't think it is necessary .We shall return only the Status Message

File: api/swagger/swagger.yaml [hunk 2]
```diff
@@ -20,7 +20,124 @@ produces:
...
+            type: array
+            items:
+              $ref: '#/definitions/UpdateEntity'
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Entity'
```
Comment (ShilpaGhanashyamGore) [inline api/swagger/swagger.yaml]: Just the status message is enough , we are not returning any other values here.

File: api/swagger/swagger.yaml (hunk 1 above)
Comment (kordianbruck) [reply api/swagger/swagger.yaml]: Generally one can do both. Returning the updated data is nice tho.

Comment (felixschorer) [reply api/swagger/swagger.yaml]: That's a GET request. Of course you have to return the requested data. Am I missing something?

**Comment by felixschorer** [comment]
@ShilpaGhanashyamGore it's probably easier to just clone the branch and run `$ swagger project edit` as it should be a lot more human readable.

File: api/swagger/swagger.yaml (hunk 1 above)
Comment (ShilpaGhanashyamGore) [reply api/swagger/swagger.yaml]: @lgfrbcsgo : Please ignore that , it was  misplaced (it was meant for PUT).
Regarding POST and PUT , as Kordian mentioned we can follow either of two patterns 
return the status message + all entities or just the status message. 
We shall then go by what we specified here - status message+ all entities.

**Review by ShilpaGhanashyamGore — COMMENTED** [review]
Reviewed only entities.js .
It is agreeable to me .

**Comment by felixschorer** [comment]
What have I done.... sorry 😒 
I didn't think it would undo the merge...

**Comment by felixschorer** [comment]
No more commits... I promise...

**Comment by ShilpaGhanashyamGore** [comment]
@lgfrbcsgo , @kordianbruck   :  I think you have to  merge to MusicConnectionMachine/api for me to do clone .

**Comment by felixschorer** [comment]
@ShilpaGhanashyamGore yeah, sorry. I accidentally undid that yesterday... It needs to be approved again to be merged.

**Comment by sacdallago** [comment]
done, branch gone

**Comment by felixschorer** [comment]
Thanks, @sacdallago

**Comment by sacdallago** [comment]
@lgfrbcsgo multitasking much?

**Comment by felixschorer** [comment]
@sacdallago always

**Pull request merged**